docs/tutorials/losses_multi_similarity.ipynb

```diff
@@ -0,0 +1,248 @@
+    "\n",
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/addons/tutorials/losses_multi_similarity\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/addons/blob/master/docs/tutorials/losses_multi_similarity.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/addons/blob/master/docs/tutorials/losses_multi_similarity.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/addons/docs/tutorials/losses_multi_similarity.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "This notebook will demonstrate how to use the MultiSimilarityLoss function in TensorFlow Addons.\n",
+    "\n",
+    "### Resources:\n",
+    "* [Multi-Similarity Loss with General Pair Weighting\n",
+    "for Deep Metric Learning](http://openaccess.thecvf.com/content_CVPR_2019/papers/Wang_Multi-Similarity_Loss_With_General_Pair_Weighting_for_Deep_Metric_Learning_CVPR_2019_paper.pdf)\n",
+    "* [Tensorflow Implementation of Multi-Similarity Loss with General Pair Weighting for Deep Metric Learning](https://github.com/geonm/tf_ms_loss)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## MultiSimilarityLoss\n",
+    "\n",
+    "MultiSimilarityLoss is implemented in two iterative steps (i.e., mining and weighting). This allows it to fully consider three similarities for pair weighting, providing a more principled approach for collecting and weighting informative pairs. Finally, the proposed MS loss obtains new state-of-the-art performance on four image retrieval benchmarks, where it outperforms the most recent approaches.\n",
+    "![Multi Similarity Loss Steps](https://user-images.githubusercontent.com/20843596/80210333-32bde480-8651-11ea-8376-a010a113fffb.PNG)\n",
+    "\n",
+    "**The loss function is described as a Euclidean distance function:**\n",
+    "\n",
+    "![function](https://user-images.githubusercontent.com/20843596/80210696-ce4f5500-8651-11ea-87d1-cc01da64681b.PNG)\n",
+    "\n",
+    "Where $\\alpha$, $\\beta$, $\\lambda$ are hyper-parameters similar as in Binomial Deviance Loss and S<sub>ij</sub> is weight."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
May you review this block from the notebook?

```
try:
  %tensorflow_version 2.x
except Exception:
  pass
```

is no longer necessary as colab is defaulted tf version 2.x now

Reply:
done